redirects-transition.conf

```diff
@@ -242,6 +242,7 @@ rewrite ^/info/tips/(.*) https://www.fec.gov/updates/?update_type=tips-for-treas
 rewrite ^/pdf/nprm/(.*) https://www.fec.gov/resources/legal-resources/rulemakings/nprm/$1 redirect;
 rewrite ^/agenda/([0-9]+)/documents/(.*) https://www.fec.gov/resources/updates/agendas/$1/$2 redirect;
 rewrite ^/agenda/([0-9]+)/(.*).pdf https://www.fec.gov/resources/updates/agendas/$1/$2.pdf redirect;
+rewrite ^/agenda/agendas2002/(.*).pdf https://www.fec.gov/resources/updates/agendas/$1 redirect;
```

Comment:
@dorothyyeager Can you please make the same changes in this file as was commented here? https://github.com/fecgov/fec-proxy/pull/148#pullrequestreview-300778983

Reply:
Thanks @patphongs - I've made the changes to this file. Appreciate your help with this!